Project qualifications.md

```diff
@@ -0,0 +1,91 @@
+# Project Qualifications
+
+This document outlines what projects qualify as entries.
+The goal of this list, in short, is:
+
+**Provide a list of awesome open source projects, that**
+**try to improve the world in some fashion, to engineers**
+**who want to altruistically participate in those.**
+
+As such, every project will be evaluated based on their
+fitness for this goal.
+A project that is valid will then be assigned a rank
+based on how friendly it is for engineers to participate
+in.
+
+## Which projects are allowed in the first place?
+
+In order to provide a sensible list, every project has to
+fulfill the following conditions in order to be added:
+
+### 1. The project must accept contributions
+
+This is rather obvious:
+If it isn't accepting code from other users, there is no
+point in listing it here.
+
+### 2. The project's goal must be the improvement, not money
+
+While it is fine, if the project is backed by a for-profit
+organisation, the project itself should not be there to make
+more money for it.
+Any earning from it should be responsibly used to further
+the developement of their solution.
+
+The visitors of this list want to improve the world, not
+make some organisation more money in their free time.
+
+### 3. The project's goal must try to improve the lives of people
+
+Previously, it was only defined that it should try to improve
+some aspect.
+However, that aspect should relate in some way to the improvements
+of lives.
+
+This can be through tackling problems like climate change, poverty,
+education, human rights or many more.
+For example, anything that can be put into one or more
+[SDGs](https://www.undp.org/content/undp/en/home/sustainable-development-goals.html)
+(Social Developement Goal) is fair game.
+
+### 4. The project must treat contributors with respect
+
+OS contributors engage in good faith and in their free time.
+When choosing a project from this list, they should be able
+to expect a welcoming and positive response by the project
+itself.
+
+Any form of racism, sexism or other kind of hostility will
+disqualify a project from this list.
```

Comment:
I'm not sure how we'll be able to evaluate that.
Do this kind of OSS project even exist?

Reply:
I'd hope they are rare. I didn't plan on trying to evaluate every project based on this, but rather just have it in case we need it.
I thought, that for any project to be on a list for potential contributors, they should also have a welcoming atmosphere.

I, honestly, don't have much experience in contributing to OSS, so, if you think, it's a useless point, I'll remove it.